### ISSUE TITLE
record type is SSL3_AL_FATAL. [desctioption: 40]

### ISSUE THREAD

**Issue by oalders** [hirose31/Net-SSL-ExpireDate#14]
Hi. Thank you for this module! I've been using it just a bit and it's not clear to me why I'm getting the error below.

```bash
perl -MNet::SSL::ExpireDate -e 'print $Net::SSL::ExpireDate::VERSION'
1.23

perl -MDevel::Confess -MNet::SSL::ExpireDate -e "Net::SSL::ExpireDate->new( https => 'webercufflinks.com')->expire_date"
record type is SSL3_AL_FATAL. [desctioption: 40] at /Users/olafalders/.plenv/versions/5.36.0/lib/perl5/site_perl/5.36.0/Net/SSL/ExpireDate.pm line 202.
        Net::SSL::ExpireDate::_peer_certificate("webercufflinks.com", 443, undef, undef) called at /Users/olafalders/.plenv/versions/5.36.0/lib/perl5/site_perl/5.36.0/Net/SSL/ExpireDate.pm line 98
        eval {...} called at /Users/olafalders/.plenv/versions/5.36.0/lib/perl5/site_perl/5.36.0/Net/SSL/ExpireDate.pm line 98
        Net::SSL::ExpireDate::expire_date(Net::SSL::ExpireDate=HASH(0x7fdea800adf8)) called at -e line 1
 at /Users/olafalders/.plenv/versions/5.36.0/lib/perl5/site_perl/5.36.0/Net/SSL/ExpireDate.pm line 99.
        Net::SSL::ExpireDate::expire_date(Net::SSL::ExpireDate=HASH(0x7fdea800adf8)) called at -e line 1
```

I don't know if this is related to #5?

**Comment by hirose31** [comment]
If possible, I would like you to try this patch fe9fa85700b1f248e4fbb8f13b1b682a27a9c5c1 or the latest master.

I will release the next version within a few days of refactoring.

**Comment by oalders** [comment]
Thanks, @hirose31! That fixes the issue for me. 😄

**Comment by hirose31** [comment]
I've just released v1.24. It will be available soon.

**Comment by oalders** [comment]
Thank you so much for this. I looked at your GitHub sponsor profile and I notice you only have the one option. If you had, for instance, a one-time "thank you" tier, that's something I'd happily donate to. You should be able to allow arbitrary amounts and set a minimum etc.
